assets/terraform-modules/aws/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -5,7 +5,11 @@ systemd:
       enable: true
```

Comment:
The commit title and message are no longer accurate. It is applied on all platforms.